File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/dotero-dev.py
for i in range (100):
    if (i+1)%3 == 0 and (i+1)%5==0:
        print("fizzbuzz")
    elif (i+1)%3 == 0:
        print ("fizz")
    elif (i+1)%5 == 0:
        print("buzz")
    else:
        print(i+1)

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/kotlin/dotero-dev.kt
fun main(){
    fizzBuzz()
}

fun fizzBuzz(){
    for (i in 1..100){
    	if (i%3==0 && i%5==0){
            println("fizzbuzz")
        } else if (i%3==0){
            println("fizz")
        } else if (i%5==0){
            println("buzz")
        } else{
            println(i)
        }
    }
}